classes/SortingProcessor.js

```diff
@@ -0,0 +1,9 @@
+module.exports = class SortingProcessor {
+  generateSortingQuery ({ sortingColumns, asc = true, defaultValue = [] }) {
```

Comment:
kind of confusing - `sortingColumns` is two word, while `asc` is shortened. Stylewise seems better to call them `columns` and `ascending`